Podfile

```diff
@@ -11,5 +11,5 @@ target 'Decred Wallet' do
   pod 'QRCodeReader.swift'
   pod 'Fabric'
   pod 'Crashlytics'
-
+  pod 'Signals'
```

Comment:
I've only seen one use case for signals in this PR, and I added a comment there. I believe we can do without it.

Reply:
Signals is used to once yes, but it handles tap event actions from our custom tab menu.